docs/src/main/asciidoc/artemis-jms-guide.adoc

```diff
@@ -0,0 +1,287 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using Artemis JMS extension
+include::./attributes.adoc[]
+
+This guide demonstrates how your Quarkus application can use Artemis JMS messaging.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+* A running Artemis server, or Docker Compose to start one
+* GraalVM installed if you want to run in native mode.
+
+== Architecture
+
+In this guide, we are going to generate (random) prices in one component.
+These prices are written in an JMS queue (`prices`).
+Another component reads from the `prices` queue and stores the last price.
+The data can be fetch from a browser using a fetch button from a JAX-RS resource.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `artemis-jms` {quickstarts-tree-url}/artemis-jms[directory].
```

Comment:
OK, we will need a quickstart too then :).

Reply:
I thought that was required :)